main.js

```diff
@@ -1,22 +1,89 @@
 var $carContainer = document.querySelector('#car');
 var $car = document.querySelector('.car-size');
-var currentDirection;
+var intervalId = null;
+var topElf = 0;
+var leftShake = 0;
 
-function movedirection(key) {
+
+function rotate(key) {
   if (key === 'ArrowUp') {
     $car.className = 'car-size rotate-up';
   }
   if (key === 'ArrowLeft') {
     $car.className = 'car-size rotate-left';
   }
   if (key === 'ArrowRight') {
-    $car.className = 'car-size roate-right';
   }
   if (key === 'ArrowDown') {
     $car.className = 'car-size rotate-down';
   }
 }
 
+function startCar() {
+
+}
+
+function move(key){
+    leftShake ++;
+    $carContainer.style.left = leftShake.toString() + 'px';
+    $carContainer.style.top = topElf.toString() + 'px';
```

Comment:
you don't need to call toString on a number, concatenating it with a string already type coerces it into a string